src/main/java/seedu/module/logic/parser/FindModuleCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.module.logic.parser;
+
+import static seedu.module.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.module.logic.commands.FindModuleCommand;
+import seedu.module.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindModuleCommand object
+ */
+public class FindModuleCommandParser implements Parser<FindModuleCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindModuleCommand
+     * and returns a FindModuleCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FindModuleCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        if (trimmedArgs.isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindModuleCommand.MESSAGE_USAGE));
+        }
+
+        String[] nameKeywords = trimmedArgs.split("\\s+");
+
+        return new FindModuleCommand(nameKeywords[0]);
+    }
+
```

Comment:
Looking at the other parse methods in the various commands like add, ArgumentMultiMap and tokenize is used instead of this way of split. Although valid, maybe for consistency sake can use that?